charts/monitor/values.yaml

```diff
@@ -14,6 +19,11 @@ influxdb:
   database: "kubernetes"
   user: "user"
   password: "password"
+  persistence:
+    enabled: false
+    storageClass: standard
+    accessMode: ReadWriteOnce
+    size: 20Gi
```

Comment:
it is already in `values.yaml` file where user can change. but my idea is put that to global `values.yaml` file too, so enabling persistence you can set the storage size too